src/components/Charts/PieChartFigure.vue

```diff
@@ -0,0 +1,201 @@
+			</div>
+		</div>
+		<!-- key -->
+		<div style="width: 12rem; height: 85%;" class="tw-flex tw-flex-col tw-justify-between">
+			<ol>
+				<li
+					v-for="(slice, index) in slices.slice(pageIndex * slicesPerPage, (pageIndex + 1) * slicesPerPage)"
+					:key="index"
+					@mouseenter="activeSlice = slice"
+					@click="activeSlice = slice"
+					class="tw-flex tw-items-center"
+				>
+					<div
+						class="tw-w-2 tw-h-2 tw-mr-1 tw-rounded-full tw-flex-none"
+						:style="{ backgroundColor: slice.color }"
+					></div>
+					<span class="tw-truncate">
+						{{ slice.label }}
+					</span>
+				</li>
+			</ol>
+			<!-- paging controls -->
+			<div v-if="pageCount > 1">
+				<button
+					:disabled="pageIndex === 0"
+					class="tw-font-medium tw-p-0.5 disabled:tw-opacity-low"
+					@click="prevPage"
+				>
+					&lt;
+				</button>
+				{{ pageIndex + 1 }} / {{ pageCount }}
+				<button
+					:disabled="pageIndex === pageCount - 1"
+					class="tw-font-medium tw-p-0.5 disabled:tw-opacity-low"
+					@click="nextPage"
+				>
+					&gt;
+				</button>
+			</div>
+		</div>
+	</figure>
+</template>
+
+<script>
+import numeral from 'numeral';
+import Alea from '@/util/experiment/Alea';
+import KvLoadingPlaceholder from '~/@kiva/kv-components/vue/KvLoadingPlaceholder';
+
+// convenience function to get point on circumfrence of a given circle (from https://codepen.io/grieve/pen/xwGMJp)
+function circumPointFromAngle(cx, cy, r, a) {
+	return [
+		cx + r * Math.cos(a),
+		cy + r * Math.sin(a)
+	];
+}
+
+export default {
+	name: 'PieChartFigure',
+	props: {
+		loading: {
+			type: Boolean,
+			default: true,
+		},
+		values: {
+			type: Array,
+			default: () => [],
+		},
+	},
+	components: {
+		KvLoadingPlaceholder,
+	},
+	data() {
+		return {
+			svgSize: 32,
+			lineWidth: 5,
+			activeSlice: null,
+			pageIndex: 0,
+			slicesPerPage: 10,
+		};
+	},
+	computed: {
+		activeSlicePercent() {
+			return this.activeSlice ? numeral(this.activeSlice.percent).format('0.00%') : '';
+		},
+		radius() {
+			return (this.svgSize / 2) - (this.lineWidth / 2) - 2;
+		},
+		center() {
+			return this.svgSize / 2;
+		},
+		slices() {
+			const slices = [];
+			// center point
+			const cX = this.center;
+			const cY = cX;
+			// radius
+			const r = this.radius;
+			// starting angle
+			let start = -0.25;
+			// loop through each value and create a pie slice path
+			for (let i = 0; i < this.values.length; i += 1) {
+				const value = this.values[i];
+				const end = start + value.percent;
+				const [startX, startY] = circumPointFromAngle(cX, cY, r, start * Math.PI * 2);
+				const [endX, endY] = circumPointFromAngle(cX, cY, r, end * Math.PI * 2);
+				const largeArc = value.percent > 0.5 ? 1 : 0;
+				// Draw just the outer arc of the slice
+				const path = `M ${startX} ${startY} A ${r} ${r} 0 ${largeArc} 1 ${endX} ${endY}`;
+				// Draw the entire slice
+				// const path = `M ${cX} ${cY} L ${startX} ${startY} A ${r} ${r} 0 ${largeArc} 1 ${endX} ${endY} Z`;
```

Comment:
Remove?

Reply:
I had left it in there in case we did want to draw a full slice, but I think that's probably easy enough to figure out again in the future if we do want to change the design. I'll pull it out to avoid confusion.